docs/modules/operation/pages/events/sources/kafka.adoc

```diff
@@ -0,0 +1,50 @@
+
+[[ga-events-sources-kafka]]
+
+= Kafka Consumer (Events)
+
+Events can also be created by routing them to a specific topic on Kafka.
+The Kafka Consumer feature allows events to be consumed from specific topic on Kafka.
```

Comment:
The Kafka Consumer feature allows events to be consumed from a specific topic on Kafka.

Reply:
```suggestion
The Kafka Consumer feature allows events to be consumed from a specific topic on Kafka.
```